src/CSAccounting.sol

```diff
@@ -108,7 +108,15 @@ contract CSAccounting is
         LIDO.approve(LIDO_LOCATOR.burner(), type(uint256).max);
     }
 
-    function finalizeUpgradeV2() external reinitializer(2) {}
+    function finalizeUpgradeV2(
+        uint256[2][] calldata defaultBondCurve,
+        uint256[2][] calldata vettedBondCurve
+    ) external reinitializer(2) {
+        CSBondCurve.__addBondCurvesWithIntervals(
+            defaultBondCurve,
+            vettedBondCurve
+        );
```

Comment:
I think the temporary method is redundant here. Consider removing it and using explicit calls.